New note for exemption property

mattgarrish opened the pull request:
Here's a first draft of a possible WG note to define the property and the three EAA values.

All feedback welcome at this stage.

Fixes #2570 

[Preview](https://cdn.statically.io/gh/w3c/epub-specs/a11y/exempt-property/epub33/a11y-exemption/index.html)

mattgarrish committed:
First draft of property

clapierre reviewed (approved):
This looks awesome Matt, well done!

mattgarrish committed:
Make title more descriptive

mattgarrish committed:
remove sentence per George's feedback

mattgarrish committed:
update respec group and mailing list metadata to pm wg

mattgarrish committed:
updated values and examples per Gregorio's feedback

mattgarrish committed:
remove refines attribute;
update the extends field to clarify the property refines the publication;
add note that a conformsTo statement isn't required

mattgarrish committed:
match wording to EAA terms

mattgarrish committed:
add editor

mattgarrish committed:
add appendix class to the property definition section
